css/.scss-lint.yml

```diff
@@ -265,6 +290,7 @@ linters:
     PropertySortOrder:
         enabled: true
         ignore_unspecified: false
+        min_properties: 1
```

Comment:
The default for `min_properties` is 2, which kinda makes sense since sorting isn’t necessary for one-rule declarations. Maybe this should stay at `2`?